client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -0,0 +1,92 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import Card from 'components/card';
+import CTACard from './cta-card';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class SelectBusinessType extends Component {
+	static propTypes = {
+		translate: PropTypes.func.isRequired,
+		siteId: PropTypes.string.isRequired,
+		recordTracksEvent: PropTypes.func.isRequired,
+	};
+
+	onCreateMyListingClick = () => {
+		this.props.recordTracksEvent( 'calypso_google_my_business_create_my_listing_click' );
+	};
+
+	onOptimizeYourSEOClick = () => {
+		this.props.recordTracksEvent( 'calypso_google_my_business_optimize_your_seo_click' );
+	};
+
+	render() {
+		const { translate, siteId } = this.props;
+
+		return (
+			<div className="select-business-type">
+				<HeaderCake
+					isCompact={ false }
+					alwaysShowActionText={ false }
+					backHref={ '/stats/day/' + siteId }
+				>
+					{ translate( 'Google My Business' ) }
+				</HeaderCake>
+				<Card className="select-business-type__explanation">
+					<div className="select-business-type__explanation-main">
+						<h1>{ translate( 'Which type of business are you?' ) }</h1>
+						<h2>
+							{ translate(
+								'Google My Business lists your local business on Google Search and Google Maps. ' +
+									'It works for businesses that have a physical location or serve a local area.'
+							) }
+						</h2>
+					</div>
+					<img
+						src="/calypso/images/google-my-business/business-local.svg"
+						alt="Local business illustration"
+					/>
+				</Card>
+
+				<CTACard
```

Comment:
What do you think of using a name less generic such as `BusinessType` or `BusinessTypeSection` instead of `CTACard`?

Reply:
Since it has nothing to do with Google My Business really, `Call To Action Card` is a good name IMO
Maybe we should extract that to live under `components/`

Reply:
I see your reasoning now. Yes moving it to `blocks` or ``components` probably make sense, although I'm not a fan of using acronyms.